python/ray/tests/test_function_manager.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import os
+import pytest
+
+import ray
+from ray._private.function_manager import FunctionActorManager
+
+
+@ray.remote
+def wrapped_func():
+    return "ok"
+
+
+def normal_func():
+    return "ok"
+
+
+def test_load_remote_function(shutdown_only):
```

Comment:
Can you move this two tests to python/ray/tests/test_basic_2.py? I found there already have a test name `test_load_code_from_local`. Maybe we can put them together and make sure your new tests will be triggered.

Reply:
moved, thanks.